bears/java/CheckstyleBear.py

```diff
@@ -75,9 +82,13 @@ def create_arguments(
         check_invalid_configuration(
             checkstyle_configs, use_spaces, indent_size)
 
-        if checkstyle_configs in known_checkstyles:
+        if checkstyle_configs in _checkstyle_provided_styles:
```

Comment:
but doing it this way makes it easier to update the dictionaries if checkstyle gets updated no?